source/_integrations/godice.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: GoDice
+description: Integrate GoDice
+ha_category:
+  - Sensor
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@anton-ptashnik'
+ha_domain: godice
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+ha_release: 0.001
```

Comment:
I need someone to tell me which version to put here

Reply:
@LennP maybe you can please

Reply:
I would just put 2024.04 for now which is the next version, if it does not make that you can update it to the next version